pkg/controller/node_test.go

```diff
@@ -55,8 +55,38 @@ func TestNodeIsReady(t *testing.T) {
 			in:  makeNode(map[string]string{"foo": "bar", "master": "true"}, false),
 			out: true,
 		},
+		{
+			in:  makeNode(map[string]string{"foo": "bar", "master": "true"}, false),
+			out: true,
+		},
+		{
+			in:             makeNode(map[string]string{"foo": "bar"}, true),
+			out:            true,
+			readinessLabel: map[string]string{},
+		},
+		{
+			in:             makeNode(map[string]string{"foo": "bar"}, false),
+			out:            false,
+			readinessLabel: map[string]string{},
+		},
+		{
+			in:             makeNode(map[string]string{readyLabel: readyValue}, false),
+			out:            false,
+			readinessLabel: map[string]string{},
+		},
+		{
+			in:             makeNode(map[string]string{"foo": "bar", "master": "true"}, false),
+			out:            true,
+			readinessLabel: map[string]string{},
+		},
 	}
 	for _, tt := range testTable {
+		if tt.readinessLabel != nil {
+			nodeTestController.opConfig.NodeReadinessLabel = tt.readinessLabel
+		} else {
+			nodeTestController.opConfig.NodeReadinessLabel = map[string]string{readyLabel: readyValue}
+		}
```

Comment:
```suggestion
		nodeTestController.opConfig.NodeReadinessLabel = tt.readinessLabel
```
I would prefer to define readinessLabel for each test to keep this part here short. Maybe we don't need all 8 combinations, e.g. when the node is scheduable

Reply:
done